catalog/spatial/geocoding/spatial-geocoding-offline-catalog/src/main/java/org/codice/ddf/spatial/geocoding/index/GeoNamesCatalogIndexer.java

```diff
@@ -360,4 +363,26 @@ private void fileProcessingComplete(String resource) {
       LOGGER.debug("Unable to create {} to indicate {} processed", processedIndicator, resource, e);
     }
   }
+
+  private long getSortPopVal(long popSize) {
+    if (popSize > 400_000) {
+      return GeoCodingConstants.POPULATION_OVER_400K;
+    } else if (popSize > 350_000 && popSize <= 400_000) {
+      return GeoCodingConstants.POPULATION_LT_400K_GT_350K;
+    } else if (popSize > 300_000 && popSize <= 350_000) {
+      return GeoCodingConstants.POPULATION_LT_350K_GT_300K;
+    } else if (popSize > 250_000 && popSize <= 300_000) {
+      return GeoCodingConstants.POPULATION_LT_300K_GT_250K;
+    } else if (popSize > 200_000 && popSize <= 250_000) {
+      return GeoCodingConstants.POPULATION_LT_250K_GT_200K;
+    } else if (popSize > 150_000 && popSize <= 200_000) {
+      return GeoCodingConstants.POPULATION_LT_200K_GT_150K;
+    } else if (popSize > 100_000 && popSize <= 150_000) {
+      return GeoCodingConstants.POPULATION_LT_150K_GT_100K;
+    } else if (popSize > 50_000 && popSize <= 100_000) {
+      return GeoCodingConstants.POPULATION_LT_100K_GT_50K;
+    } else {
+      return GeoCodingConstants.POPULATION_UNDER_50K;
+    }
+  }
```

Comment:
could replace this with roughly `return Math.max(2, Math.min(11, popSize/40000))`

Reply:
Not a bad idea at all. That's much better

Reply:
I think this does it exactly `Math.max(10, popSize/50_000 + 2)`